plugins/languages/treesitter/treesitter.nix

```diff
@@ -12,6 +12,103 @@ helpers.neovim-plugin.mkNeovimPlugin config {
   luaName = "nvim-treesitter.configs";
   defaultPackage = pkgs.vimPlugins.nvim-treesitter;
 
+  description = ''
+    Provides an interface to [tree-sitter]
```

Comment:
```suggestion
```

Reply:
@menixator Sorry to flip on this, I didn't realize it would hotlink to actual tree-sitter and not the plugin again. I think it's safe to add this back.

Reply:
Afterwards, can you squash all the commits into a single one.

Reply:
@khaneliman done.